src/components/Search/Search.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { NavLink, Redirect } from 'react-router-dom';
+import './Search.css';
+import { BiSearch } from 'react-icons/bi';
+
+class Search extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchTerm: '',
+      isActive: false
+    }
+  }
+
+  handleClick = () => {
+    if (this.state.searchTerm) {
+      this.props.search(this.state.searchTerm);
+    }
+    this.setState({isActive: !this.state.isActive});
+    this.clearInput();
+  }
+
+  handleChange = event => this.setState({searchTerm: event.target.value});
+
+  handleKeyDown = event => event.key === 'Enter' && this.handleClick();
```

Comment:
We still need to fix the bug where enter isn't always working. I'll make an issue for it.